source/features/widgets/circular-progress/components/CircularProgress.tsx

```diff
@@ -25,11 +25,11 @@ const CircularProgress = (props: ICircularProgressProps) => {
   let percentageCircleStyle = null;
   if (percentage <= 50) {
     percentageCircleStyle = {
-      backgroundImage: `linear-gradient(238deg, transparent 50%, #404250 50%), linear-gradient(${rotateDegree}deg, #1fc1c3 50%, #404250 50%)`,
+      backgroundImage: `linear-gradient(238deg, transparent 50%, #404250 50%), linear-gradient(${rotateDegree}deg, #eb2256 50%, #404250 50%)`,
```

Comment:
@DeeJayElly I made this component using static color before themes were setup.
Now we have themes setup, style variables are there.
So we need to use color style variables here to apply styles to this `Circular Progress` component.
Seems like we should have style variables in js code as well as scss to use them in such case.
You can implement it here or if it takes a bit long, we can create another ticket or postpone that to next round.

Reply:
Lets postpone this to another ticket.